src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -163,6 +163,22 @@ Namespace Microsoft.CodeAnalysis.Operations
                 isImplicit:=isImplicit OrElse argument Is Nothing)
         End Function
 
+        Private Function CreateReceiverOperation(node As BoundNode, symbol As ISymbol) As IOperation
+            If node Is Nothing OrElse node.Kind = BoundKind.TypeExpression Then
+                Return Nothing
+            End If
+
+            If symbol IsNot Nothing AndAlso
+               node.WasCompilerGenerated AndAlso
+               symbol.IsStatic AndAlso
+               (node.Kind = BoundKind.MeReference OrElse
+                node.Kind = BoundKind.WithLValueExpressionPlaceholder) Then
```

Comment:
>WithLValueExpressionPlaceholder [](start = 38, length = 31)

If we are handling this node, it feels like we should also handle WithRValueExpressionPlaceholder.

Reply:
It is used by With statement.

---
In reply to: [149470915](https://github.com/dotnet/roslyn/pull/22972#discussion_r149470915) [](ancestors = 149470915)

Reply:
I've added a test for this, but that actually already doesn't appear to have an instance receiver. We shouldn't need to filter it out.

---
In reply to: [149471429](https://github.com/dotnet/roslyn/pull/22972#discussion_r149471429) [](ancestors = 149471429,149470915)

Reply:
> I've added a test for this, but that actually already doesn't appear to have an instance receiver. We shouldn't need to filter it out.

I don't see a test for a With **statement** which uses an RValue as a target.

---
In reply to: [149534995](https://github.com/dotnet/roslyn/pull/22972#discussion_r149534995) [](ancestors = 149534995,149471429,149470915)

Reply:
I seem to learn something new about VB every day. Added IWithStatement_StaticFieldReferenceAsRValue_NoInstanceReceiver, and added WithRValueExpressionPlaceholder as an exception here.

---
In reply to: [149542018](https://github.com/dotnet/roslyn/pull/22972#discussion_r149542018) [](ancestors = 149542018,149534995,149471429,149470915)